libagbsyscall/libagbsyscall.c

```diff
@@ -146,11 +148,12 @@ void RegisterRamReset(u32 flags)
     }
 
     if (flags & RESET_SIO_REGS) {
-        memset(REG_SIOCNT, 0, ((REG_SIOMULTI3 + 1) - REG_SIOCNT));
+        memset((void *)(int)REG_SIOCNT, 0, (int)((REG_SIOMULTI3 + 1) - REG_SIOCNT));
     }
 
     if (flags & RESET_SOUND_REGS) {
-        memset(REG_SOUND1CNT_L, 0, ((REG_FIFO_B + 1) - REG_SOUND1CNT_L));
+        memset((void *)(int)REG_SOUND1CNT_L, 0,
+               (int)((REG_FIFO_B + 1) - REG_SOUND1CNT_L));
     }
```

Comment:
The respective first casts don't have to be (void *)(int), they can just be (void *) instead, right?

Reply:
I can't remember why I did it this way